Formula/terraforming.rb

```diff
@@ -0,0 +1,20 @@
+class Terraforming < Formula
+  desc "Export existing AWS resources to Terraform style (tf, tfstate)"
+  homepage "http://terraforming.dtan4.net/"
+  url "https://github.com/dtan4/terraforming.git",
+      :tag => "v0.14.0",
+      :revision => "5e934a1a8d18062f9c4e4e3f4a52dd648fc9976d"
+  head "https://github.com/dtan4/terraforming.git"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "terraforming.gemspec"
+    system "gem", "install", "terraforming-0.14.0.gem"
+    bin.install libexec/"bin/terraforming"
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+  end
+
+  test do
+    system "#{bin}/terraforming", "--help"
```

Comment:
The test needs to exercise and verify the functionality of the software.

Reply:
Testing more features of this software would imply connecting to an existing AWS account, except asserting the output of the `version` command I don't really know what kind of test can be performed.

Reply:
You can also check for an anticipated failure case.